cmds/tui.c

```diff
@@ -182,11 +182,11 @@ static const char *graph_field_names[NUM_GRAPH_FIELD] = {
 	"ADDRESS",
 };
 
-#define NUM_REPORT_FIELD 10
+#define NUM_REPORT_FIELD 12
 
 static const char *report_field_names[NUM_REPORT_FIELD] = {
-	"TOTAL TIME", "TOTAL AVG", "TOTAL MIN", "TOTAL MAX", "SELF TIME",
-	"SELF AVG",   "SELF MIN",  "SELF MAX",	"CALL",	     "SIZE",
+	"TOTAL TIME", "TOTAL AVG", "TOTAL MIN", "TOTAL MAX", "TOTAL STD", "SELF TIME",
+	"SELF AVG",   "SELF MIN",  "SELF MAX",	"SELF STD",  "CALL",	  "SIZE",
```

Comment:
Maybe "STD" looks confusing, how about "STDV"?

Reply:
Or just "SD" according to the [wikipedia](https://en.wikipedia.org/wiki/Standard_deviation).

Reply:
Thanks for the suggestion! I'll change the name to STDV here